src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -430,7 +431,7 @@ export function HeaderRow() {
 }
 
 /* Loading State: row component with loading bubbles */
-export function LoadingRow() {
+export function LoadingRow({ style }: { style?: CSSProperties }) {
```

Comment:
When setting up passthroughs like this I generally try to do something like `{...props}` on the styled child instead of individually defining the props here.

Reply:
bumping this

Reply:
done